.github/workflows/pr-test.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
           - test: no-changes
             allow-failure: false
           - test: format-error
-            allow-failure: true
+            allow-failure: false 
```

Comment:
Our of curiosity, why did this test change?

Reply:
Why you ask? Pure incompetence.... 🫠 

I did not realize yesterday that the autoformat was formatting the code and therefore passing the test. This is completely due to my misunderstanding of the pr-test yaml workflow

Reply:
Totally fair - it is a hacky bit of code trying to run integration tests on the action from the workflow results.

Reply:
I will change it back though since I did not know what I was doing and I made it worse.